patterns/footer-writer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Title: Footer writer
+ * Slug: twentytwentyfour/footer-writer
+ * Categories: footer
+ * Block Types: core/template-part/footer
+ */
+?>
+<!-- wp:group {"tagName":"footer","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|50"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"center"}} -->
+<footer class="wp-block-group" style="padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--50)">
+
+<!-- wp:site-logo /-->
+
+<!-- wp:navigation {"overlayMenu":"never","layout":{"type":"flex","justifyContent":"center"},"fontSize":"small"} /-->
+
+<!-- wp:paragraph {"align":"center","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}}},"textColor":"secondary","fontSize":"small"} -->
+<p class="has-text-align-center has-secondary-color has-text-color has-link-color has-small-font-size">Designed with <a href="http://wordpress.org" data-type="link" data-id="wordpress.org">WordPress</a></p>
```

Comment:
Same comment as above about making this one [translatable](https://github.com/WordPress/twentytwentyfour/blob/trunk/patterns/footer.php#L17-L23).